src/gui/widgets/LedCheckbox.cpp

```diff
@@ -56,13 +52,10 @@ LedCheckBox::LedCheckBox( const QString & _text, QWidget * _parent,
 
 LedCheckBox::LedCheckBox( QWidget * _parent,
 				const QString & _name, LedColors _color ) :
-	DEFAULT_LEDCHECKBOX_INITIALIZER_LIST
+	LedCheckBox( QString::null, _parent, _name, _color)
```

Comment:
I'm not 100% sure about `QString::null` here. Is that okay or should that be `NULL` or something else? It's been a while since I did some C++ :) the macro left the member uninitialized when calling this constructor.

Reply:
Before this change, `m_text` has been constructed with its default ctor `QString::QString()`, so I'd simply pass `QString()` here:

```suggestion
	LedCheckBox( QString(), _parent, _name, _color )
```

Reply:
Done, replaced with the default constructor